src/apify_error.js

```diff
@@ -16,4 +18,18 @@ export default class ApifyClientError extends Error {
 
         Error.captureStackTrace(this, ApifyClientError);
     }
+
+    toString() {
+        let details = '';
+
+        if (this.details) {
+            const items = [];
+            _.mapObject(this.details, (val, key) => {
+                items.push(`${key}=${JSON.stringify(val)}`);
+            });
+            details = `(${items.join(', ')})`;
+        }
+
+        return `[${this.name}] ${APIFY_ERROR_NAME}: ${this.message} ${details}`;
```

Comment:
Why no stack?

Reply:
good point, maybe we should include it too